Can't do SCU build on Windows

### Tested versions

Regression from a5cf92664db9633b9fb1307a801d99de70d09aba

### System information

Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1060 (NVIDIA; 31.0.15.4633) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

Compiling SCU build results in
```
LINK : fatal error LNK1181: cannot open input file „modules\module_gdscript.windows.editor.dev.x86_64.lib”
```
CC @Riteo 

### Steps to reproduce

`scons dev_build=yes scu_build=yes`

### Minimal reproduction project (MRP)

N/A

EDIT:
Seems like reverting `scu_builders.py` changes is enough to fix this.

--- a/scu_builders.py
+++ b/scu_builders.py
@@ -23,8 +23,9 @@ def clear_out_stale_files(output_folder, extension, fresh_files):
         return
 
     for file in glob.glob(output_folder + "/*." + extension):
+        file = Path(file)
         if not file in fresh_files:
-            # print("removed stale file: " + file)
+            # print("removed stale file: " + str(file))
             os.remove(file)
 
 
@@ -97,7 +98,7 @@ def write_output_file(file_count, include_list, start_line, end_line, output_fol
     elif _verbose:
         print("SCU: Generation not needed for: " + short_filename)
 
-    return output_filename
+    return output_path
 
 
 def write_exception_output_file(file_count, exception_string, output_folder, output_filename_prefix, extension):
@@ -124,7 +125,7 @@ def write_exception_output_file(file_count, exception_string, output_folder, out
     elif _verbose:
         print("SCU: Generation not needed for: " + short_filename)
 
-    return output_filename
+    return output_path
 
 
 def find_section_name(sub_folder):

